internal/pkg/action/operator_uninstall.go

```diff
@@ -190,3 +193,12 @@ func contains(haystack []string, needle string) bool {
 	}
 	return false
 }
+
+func isIPWithInstalledCSV(ip v1alpha1.InstallPlan, installedCSV string) bool {
+	for _, step := range ip.Status.Plan {
+		if step.Resource.Kind == csvKind && step.Resource.Name == installedCSV {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
For now, I think we should check the install plan's `spec.ClusterServiceVersionNames`, and select an install plan only if it is length 1 and is equal to the CSV name we're looking for.

Reply:
And maybe return an error if we find our csv name, but others are listed.

Reply:
Have modified the implementation. By `others are listed`, do you mean when we find an installplan which has multiple `ClusterServiceNames` - among which is the installed CSV, we return an error because we may have dependent resources specified in other csvs (maybe of another operator) ?

Reply:
Yep, exactly!

Reply:
Got it! Will modify it.